cookbook_se/recipes/core/intermediate/run_conditions.py

```diff
@@ -107,3 +112,34 @@ def multiplier_3(my_input: float) -> float:
 
 
 print(f"Output of multiplier_3(my_input=5): {multiplier_3(my_input=5)}")
+
+
+# %%
+# Example 4
+# ^^^^^^^^^^
+#
+# It is possible to test if a boolean retruned from previous tasks is True or False. But, Unary operations are not supported in Flytekit. To achieve this users can use helpful methods like is_true, is_false or is_ on the output variable.
+#
+# .. note::
+#
+#    Wondering how output values get these methods. In a workflow no output value is available to access directly. The inputs and outputs are auto-wrapped in a special object called :ref:pyclass:`flytekit.annotated.promise.Promise`.
+#
+@task
+def return_true() -> bool:
+    return True
+
+
+@workflow
+def failed() -> int:
+    return 10
+
+
+@workflow
+def success() -> int:
+    return 20
+
+
+@workflow
+def decompose() -> int:
+    result = return_true()
+    return 10  # conditional("test").if_(result.is_true()).then(success()).else_().then(failed())
```

Comment:
should this be uncommented?

Reply:
should this entire section be in here?  this looks like joseph's example.